components/read_your_meter.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "read_your_meter",
+  "owner": ["smhgit"],
+  "manifest": "https://https://raw.githubusercontent.com/eyalcha/read_your_meter/master/custom_components/read_your_meter/manifest.json",
```

Comment:
That URL seems wrong...?

Reply:
typo, https was duplicated. fixed

Reply:
@frenck are there any changes I need to do? I am not sure as it shows "Changes requested" but on the other hand all checks have passed.

Reply:
I was awaiting some builds and changes upstream. This PR has potentially issue with building on our end. Hence I was waiting for everything to be clear at our end).